### DISCUSSION TITLE
listbackup does not list backups with custom names

### DISCUSSION THREAD

**Discussion by JeanMarie-PM** [Archmonger/django-dbbackup#586] [Q&A] [answered]
# Bug Report

_Please help us help you by filling out any applicable information in this template and removing the rest!_

## Describe the bug

listbackup shows backups created without a -o option, but does not show ones created with this option
media restore with a -i p[tion works as expected.
## To Reproduce
`./manage.py mediabackup -o mymedia.tar
`INFO Writing file to mymedia.tar

`./manage.py listbackups `
Name                                     Datetime            
53274d44972d-2023-09-26-182031.tar       09/26/23 18:20:31   
default-53274d44972d-2023-09-26-181511.psql.bin 09/26/23 18:15:11   
default-53274d44972d-2023-09-26-181803.psql.bin 09/26/23 18:18:03   

## Expected behavior

Expect to see mymedia.tar in the output

## Screenshots or reproduction

If applicable, add screenshots (errors, example of the behavior, etc.) to help explain your problem or post a link to a repository that replicates the issue.

### Django-dbbackup
- pypi: [e.g. 3.3.0]

### External tools

- Python: [e.g. 3.7]
- Django: [e.g. 2.2.0]
- OS: [Linux, MacOS, Windows]

**Comment by BasiliskHill** [comment, marked as answer]
This occurs because of how we process the files in the backup directory. At the moment, only files that are marked with the same date format provided in the `DBBACKUP_DATE_FORMAT` setting. It should technically be possible to add a setting or a command option to show the other files, but the date time field wouldn't be populated, making the management command a shortcut to list the documents in your backup directory.

The other option would logically be to apply the time format to the file name in some other way through the backup command, though we would need to make sure it's very deliberately applied and communicated with the user.
